docs/java/guides/cf-xsuaa.md

```diff
@@ -0,0 +1,231 @@
+---
+id: cf-xsuaa
+title: SAP Cloud Foundry
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Cloud Foundry XSUAA Usage
+custom_edit_url: https://github.com
+description: Configure you Cloud Foundry CLI and bind it to SAP Cloud Platform
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+- xsuaa
+- destination
+- connectivity
+image:
+---
+
+## XSUAA Usage on SCP Cloud Foundry
```

Comment:
I think we should make it more prominent that this guide is completely unrelated to the Cloud SDK. We don't want users to think that they have to do all of these things when using the SDK. 

So maybe use a headline like `Background knowledge on Authorization & Authentication`. And a prominent note that the Cloud SDK handles all these things and this is  mostly FYI.